README.md

```diff
@@ -0,0 +1,84 @@
+# LBRY Block Explorer
+
+A simple PHP block explorer for browsing transactions and claims on the [LBRY](https://lbry.io) blockchain. The explorer was developed using CakePHP which is a model-view-controller (MVC) PHP framework.
+
+## Installation
+There are some prerequisites that need to be installed before the explorer can be accessed.
+* Web server - Apache, caddy or nginx
+* lbrycrd with txindex turned on
+* Python claims decoder (https://github.com/cryptodevorg/lbry-decoder)
+* MariaDB 10.2 or higher
+* Redis Server (optional, only required for the CakePHP redis cache engine, or to run `forevermempool`)
+* PHP 7.2 or higher
+  * php-fpm
+  * igbinary extension (https://github.com/igbinary/igbinary)
+  * phpredis extension (https://github.com/phpredis/phpredis)
+* composer (PHP package manager)
+
+### Installation steps
+* Clone the Github repository. `git clone https://github.com/lbryio/block-explorer`
+* Create a MariaDB database using the DDL found in `block-explorer/sql/lbryexplorer.ddl.sql`
+* Change the working directory to the cloned directory and run composer.
+```
+cd block-explorer
+composer update
+```
+* Create the directories, `tmp` and `logs` in the `block-explorer` folder if they have not been created yet, and make sure that they are writable by the web server.
+* Copy `config/app.default.php` to `config/app.php`. Edit the database connection values to correspond to your environment.
+* Copy `config/lbry.default.php` to `config/lbry.php`. Update the values for LBRY RPC URL and the Redis URL to correspond to your environment.
+* Configure your web server with the host root folder set to `<path to>/block-explorer/webroot` where `<path to>` is the absolute path to the configuraiton. Here is a sample nginx configuration. You can make changes to this configuration to correspond to your environment.
+```
+server {
```

Comment:
This could probably be checked-in rather than inlined? Another point I'm ambivalent about

Reply:
Usually makes sense to have sample configuration files with the actual configuration files ignored by git, so that subsequent edits with actual credentials don't get pushed to the repository.